client/my-sites/domains/domain-management/settings/index.tsx

```diff
@@ -137,6 +139,7 @@ const Settings = ( {
 	const renderStatusSection = () => {
 		if (
 			! ( domain && selectedSite?.options?.is_domain_only ) ||
+			sites.length === sites.filter( ( site ) => site?.options?.is_domain_only ).length ||
```

Comment:
Re: p1690233528375229/1690224436.392089-slack-C05CT832K2T we need to also check that the user is an owner of at least one non-domain-only site.

We could filter for a site with that condition (owner && non-domain-only) and return false if it exists.

Reply:
I will duplicate the same from [here](https://github.com/Automattic/wp-calypso/blob/544283df5cb36b94de74767011dc4e5f91ed5845/client/my-sites/domains/domain-management/transfer/transfer-to-other-site/transfer-domain-to-other-site.tsx#L52)